credentials/alts/alts_test.go

```diff
@@ -44,7 +44,7 @@ import (
 )
 
 const (
-	defaultTestLongTimeout  = 10 * time.Second
+	defaultTestLongTimeout  = 60 * time.Second
```

Comment:
I'm not sure I fully understand this long timeout. It's used only for `establishAltsConnection' and has the comment `			// The server is not ready yet. Try again.` However the server should be up and running because of synchronous `startServer` call preceding the  `establishAltsConnection'. What am I missing here?

Reply:
That comment was just a bit out-of-date. I've updated it to better reflect the current state of affairs. Thanks for flagging this!